pkg/storaged/pages.jsx

```diff
@@ -802,28 +826,14 @@ export const StorageDescription = ({ title, value, action, children }) => {
 export const StoragePage = ({ location, plot_state }) => {
     const page = get_page_from_location(location);
 
-    const parent_crumbs = [];
-    let pp = page.parent;
-    while (pp) {
-        parent_crumbs.unshift(
-            <BreadcrumbItem key={pp.name} to={"#" + cockpit.location.encode(pp.location)}>
-                {page_display_name(pp)}
-            </BreadcrumbItem>
-        );
-        pp = pp.parent;
-    }
-
     return (
         <Page id="storage">
-            { parent_crumbs.length > 0 &&
+            { (!client.in_anaconda_mode() && page.parent) &&
             <PageBreadcrumb stickyOnBreakpoint={{ default: "top" }}>
-                <Breadcrumb>
-                    { parent_crumbs }
-                    <BreadcrumbItem isActive>{page_display_name(page)}</BreadcrumbItem>
-                </Breadcrumb>
+                <StorageBreadcrumb page={page} />
             </PageBreadcrumb>
             }
-            <PageSection isFilled={false}>
+            <PageSection isFilled={false} padding={client.in_anaconda_mode() ? { default: "noPadding" } : {}}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19876-20240201-080923-67c3f528-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)